lib/github_api/client/git_data/tags.rb

```diff
@@ -4,6 +4,7 @@ module Github
   class Client::GitData::Tags < API
     # This tags api only deals with tag objects -
     # so only annotated tags, not lightweight tags.
+    # Refer https://developer.github.com/v3/git/tags/#parameters
 
     VALID_TAG_PARAM_NAMES = %w[
```

Comment:
Trust me on that one :smile: Some time in the future the valid params will change(they changed a lot in the past years), and I won't know or be able to change it due to life commitments. The library henceforth will become incompatible and break for many people. I don't have time to be checking and updating valid parameters. The onus should be on the user to check that they are sending the right payload which will be validated on the server side anyways. No point mirroring what is already implemented even if it means extra trip to the box.